docs/integrations/sigrid-api-documentation.md

```diff
@@ -541,6 +505,36 @@ com/rest/analysis-results/api/v1/objectives/{customer}`) is as follows:
 
 </details>
 
+#### Retrieving objectives for a system
+
+    GET https://sigrid-says.com/rest/analysis-results/api/v1/objectives/{customer}/{system}/config
+    
+This end point returns the following response structure:
+
+    {
+      "MAINTAINABILITY": 4.0,
+      "NEW_CODE_QUALITY": 3.5,
+      "OSH_MAX_SEVERITY": "LOW",
+      "TEST_CODE_RATIO": 0.8
+    }
+
+Systems always have a single target per objective type. If you define multiple overlapping objectives, 
+precedence rules are applied to determine which objective "wins" and gets to decide the target for
+that system:
+
+- System-level objectives have precedence over portfolio-level objectives.
+- Conditional portfolio-level objectives have precedence over unconditional portfolio-level objectives.
+- Unconditional portfolio-level objectives have the lowest precedence, and therefore act like sensible
+  default policies for your portfolio.
```

Comment:
This second bullet is what we call "Principle 2". However, it has never been implemented, so I think we should delete this second bullet entirely. Here's my suggestion:
```suggestion
- System-level objectives have precedence over portfolio-level objectives.
- If the metadata of a system matches more than one portfolio objective of the same type, the portfolio objective with the strictest target "wins". 
```